### DISCUSSION TITLE
quarkus-roq/posts/do-you-want-to-publish-a-blog-post-series/

### DISCUSSION THREAD

**Discussion by giscus** [quarkiverse/quarkus-roq#308] [Comments]
# quarkus-roq/posts/do-you-want-to-publish-a-blog-post-series/

Make your blog posts part of a series.

https://pages.quarkiverse.io/quarkus-roq/posts/do-you-want-to-publish-a-blog-post-series/

**Comment by mcruzdev** [comment]
Nice feature @jtama
